k2/csrc/compose.cu

```diff
@@ -507,7 +508,7 @@ class MultiGraphDenseIntersect {
               a_fsas_arc_idx012 = a_fsas_arc_idx01x + ai_arc_idxxx2;
       Arc arc = arcs[a_fsas_arc_idx012];
 
-      int32_t scores_idx0x = b_fsas_row_ids1[ai_fsa_idx0],
+      int32_t scores_idx0x = b_fsas_row_ids1[ai_fsa_idx0], // FIXME(fangjun): this is probably b_fsa_row_splits1  // NOLINT
```

Comment:
you are right, should be the row-splits.

Reply:
Fixed.